Update error tracking docs (#10119)
PostHog/posthog.com: contents/docs/error-tracking/index.mdx

![Exception events](https://res.cloudinary.com/dmukukwp6/image/upload/Screenshot_2024_10_01_at_18_08_56_8efb2e9aba.png)

## Custom issue grouping

> PostHog attempts to group the same exceptions as a single issue. An `$exception_fingerprint` property is generated during ingestion by PostHog and used to perform this grouping. Setting the `$exception_fingerprint` property on the frontend will override the default PostHog grouping to allow for custom grouping of certain exceptions.

Author: ```suggestion
PostHog attempts to group the same exceptions as a single issue. An `$exception_fingerprint` property is generated during ingestion by PostHog and used to perform this grouping. Setting the `$exception_fingerprint` property on the frontend will override the default flow to allow for custom grouping of certain exceptions.
```

> If using the `captureException` method you can provide `$exception_fingerprint` as an additional property in the functions second argument.

Author: ```suggestion
When using the `captureException` method you can provide `$exception_fingerprint` as an additional property in the functions second argument.
```

```js
posthog.captureException(error, { $exception_fingerprint: "MyCustomGroup" })
```

> Should an exception be autocaptured you will need to modify the properties before the event is sent. The PostHog config offers a [`before_send`](https://posthog.com/docs/libraries/js#amending-events-before-they-are-captured) hook that fires for each event. You can alert the event as part of this callback:

Author: ```suggestion
Should an exception be autocaptured you will need to modify the properties before the event is sent. The PostHog config offers a [`before_send`](https://posthog.com/docs/libraries/js#amending-events-before-they-are-captured) hook that fires for each event. You can alter the event as part of this callback to add the property:
```
